front/app/modules/commercial/content_builder/admin/containers/index.tsx

```diff
@@ -1,7 +1,78 @@
 import React from 'react';
 
+// Components
+import GoBackButton from 'components/UI/GoBackButton';
+import Button from 'components/UI/Button';
+
+// Styling
+import styled from 'styled-components';
+
+const TopContainer = styled.div`
+  border-style: solid;
+  border-width: 1px;
+  border-color: Gainsboro;
+  width: 100%;
+  display: flex;
+  background: white;
+  align-items: center;
+`;
+
+const HeaderContainerLeft = styled.div`
+  padding: 15px;
+  width: 210px;
+`;
+
+const HeaderContainerRight = styled.div`
+  border-left-style: solid;
+  border-left-width: 1px;
+  border-left-color: Gainsboro;
+  padding: 15px;
+  flex-grow: 1;
+  align-items: center;
+  display: flex;
+  gap: 10px;
+`;
+
+const ProjectDescriptionContainer = styled.div`
+  flex-grow: 2;
+`;
+
+const ProjectTitle = styled.p`
+  margin-bottom: 8px;
+  color: Gray;
+`;
+
+const BuilderTitle = styled.h3`
```

Comment:
Semantically, it's better if this is h1

Reply:
@IvaKop Ok, that makes sense. The h1 looks quite different from the prototype though - would it be fine add the styling so it looks like the h3? Or should we ask to update the design?